pkg/netcat/util.go

```diff
@@ -60,3 +65,115 @@ func (sw *ConcurrentWriter) Write(p []byte) (n int, err error) {
 	defer sw.mu.Unlock()
 	return sw.writer.Write(p)
 }
+
+type ConcurrentWriteCloser struct {
+	ConcurrentWriter
+}
+
+func NewConcurrentWriteCloser(wc io.WriteCloser) *ConcurrentWriteCloser {
+	return &ConcurrentWriteCloser{ConcurrentWriter{writer: wc}}
+}
+
+func (swc *ConcurrentWriteCloser) Close() error {
+	swc.ConcurrentWriter.mu.Lock()
+	defer swc.ConcurrentWriter.mu.Unlock()
+	return swc.ConcurrentWriter.writer.(io.WriteCloser).Close()
+}
+
+// StdoutWriteCloser wraps os.Stdout such that it satisfy the io.WriteCloser
+// interface, *but* with a special Close. Per POSIX, fd#1 should always remain
+// open; therefore, we can't close os.Stdout, even temporarily. Instead, in
+// order to drop fd#1's reference to its underlying file description, duplicate
+// "/dev/null" over fd#1.
+type StdoutWriteCloser struct{}
+
+func NewStdoutWriteCloser() *StdoutWriteCloser {
+	return &StdoutWriteCloser{}
+}
+
+func (swc *StdoutWriteCloser) Write(b []byte) (n int, err error) {
+	return os.Stdout.Write(b)
+}
+
+func (swc *StdoutWriteCloser) Close() error {
```

Comment:
The commend can be as simple as Close implements io.Close or whatever, but the linter will want a comment.

Reply:
https://golangci-lint.run/welcome/install/#local-installation
and then `golangci-lint run` in u-root root directory

Reply:
We don't this particular linter, but indeed golangcilint is failing, you can run it locally:
```
go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.64.5
GOOS=linux GOOS=amd64 golangci-lint run ./... # from u-root directory
```

Reply:
@jensdrenhaus @binjip978 thanks, I've managed to install it!

Added StdoutWriteCloser comments in v2.